tests/kas_with_instance.spec.ts

```diff
@@ -177,13 +177,57 @@ test('test instance dashboard on instance name click', async ({ page }) => {
   await expect(page.getByTestId('pageKafka-tabDashboard')).toHaveCount(1);
 });
 
+// test_4kas.py test_kafka_dashboard_default
+test('test kafka dashboard default', async ({ page }) => {
+  await waitForKafkaReady(page, testInstanceName);
+  await page.locator('a', { hasText: `${testInstanceName}` }).click();
+  await expect(page.locator('h1', { hasText: `${testInstanceName}` })).toHaveCount(1);
+  await expect(page.locator('button').locator('span', { hasText: 'Dashboard' })).toHaveCount(1);
+  await expect(page.locator('h3', { hasText: 'Topics' })).toHaveCount(1);
+  await expect(page.locator('h3', { hasText: 'Topic partitions' })).toHaveCount(1);
+  await expect(page.locator('h3', { hasText: 'Consumer groups' })).toHaveCount(1);
+  await expect(page.locator('h3', { hasText: '0' })).toHaveCount(3);
+});
+
 // test_4kafka.py test_kafka_topic_create
 test('create and delete a Kafka Topic', async ({ page }) => {
   await navigateToKafkaTopicsList(page, testInstanceName);
   await createKafkaTopic(page, testTopicName);
   await deleteKafkaTopic(page, testTopicName);
 });
 
+// test_4kafka.py test_kafka_topics_opened
+test('test kafka topics opened', async ({ page }) => {
+  await waitForKafkaReady(page, testInstanceName);
+  await page.locator('a', { hasText: `${testInstanceName}` }).click();
+  await page.locator('button[aria-label="Topics"]').click();
+  await expect(page.locator('h2', { hasText: 'No topics' })).toBeVisible();
+  await expect(page.locator('button', { hasText: 'Create topic' })).toBeVisible();
+});
```

Comment:
can't we put this check to already existing test to same some time during test execution?

Reply:
I was just migrating an already existing test on IQE. wdyt @mmusil ? Do you agree to incorporate this check into another test or to keep it as a separate one?

Reply:
cc @mmusil

Reply:
Yes please, put it for example into the test `create and delete a Kafka Topic`.

Reply:
ok I move it there @mmusil